--- a/configure	2022-04-25 20:56:56.000000000 +0000
+++ b/configure	2022-06-02 14:44:41.694175784 +0000
@@ -8244,25 +8244,24 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char fts_close ();
+int fts_close (void* fts);
 int
 main ()
 {
-return fts_close ();
+return fts_close (0);
   ;
   return 0;
 }
 _ACEOF
-for ac_lib in '' fts; do
+for ac_lib in fts; do
   if test -z "$ac_lib"; then
     ac_res="none required"
   else
     ac_res=-l$ac_lib
     LIBS="-l$ac_lib  $ac_func_search_save_LIBS"
   fi
-  if ac_fn_c_try_link "$LINENO"; then :
+#  if ac_fn_c_try_link "$LINENO"; then :
   ac_cv_search_fts_close=$ac_res
-fi
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext
   if ${ac_cv_search_fts_close+:} false; then :
